### ISSUE TITLE
HMR setup

### ISSUE THREAD

**Issue by cezarneaga** [shrynx/react-super-scripts#11]
Is HMR supposed to work without any configs?
Mine doesnt seem to work. using Preact though.

**Comment by shrynx** [comment]
@cezarneaga yes, i noticed, it works with react but not with preact. I am using babel-preset-hmre for this and maybe preact isn't compatible. I checked one or two boilerplate but they had same issue. If you get a work around or find any boilerplate having a working hmr, do tell :)

**Comment by cezarneaga** [comment]
one could write the code in react and build in preact :)

**Comment by shrynx** [comment]
haha, you can. but too risky, wouldn't know if something is breaking. I will look into this soon.

**Comment by cezarneaga** [comment]
here's something that might help: https://github.com/developit/react-router-4-test
